UCSD-E4E/Pyrenote
# Hotfix V1.2.1

There was an issue of the site not scrolling down to the next/previous buttons

to test
- Before entering annotation page, make your website window about half to a quarter size
- Load annotation page, ensure all buttons are accessible
- Make an annotation on the spectrogram, observe the the label categories push the page down
- Are the next and previous buttons still accessible?
- Repeat with increasingly smaller screens until the size becomes an extreme case
- Repeat experiment by enabling and disabling reference window feature


Additionally the hotfix fixes a bug that didn't update the segmentation's last modified property. To test
- Have one user create and save an annotation
- Have a diffrent user edit the annotation and save it
- Download the json output from admin portal and see both users are listed under the last_modified_by property
